Project/Project/Services/RecommendationService.cs

```diff
@@ -0,0 +1,109 @@
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Project.Data;
+using Project.Models;
+
+namespace Project.Services
+{
+    public class RecommendationService
+    {
+        private static DataContext _context;
+        private static AiApiService _aiApiService;
```

Comment:
Remove all static keyword, use dependency injection, register the service in the start up code